velox/exec/tests/RowContainerTest.cpp

```diff
@@ -466,8 +466,40 @@ class RowContainerTest : public exec::test::RowContainerTestBase {
     }
   }
 
+  template <typename T, bool mayHaveNulls>
```

Comment:
Don't see the use of template type 'T' in the method. Is something missing ?

Reply:
No, you are right. It is not needed. Which also means I can remove it from the other test as well. It is used in callers to distinguish the double/float type but it is not needed at this level anymore.